### ISSUE TITLE
Google Chrome Portable installer failed to download required files

### ISSUE THREAD

**Issue by kiloparseck** [sandboxie-plus/Sandboxie#168]
Google Chrome Portable installer from PortableApps.com on SBie 5.43.6 failed to download required files, it seems cannot connect to the Internet.

![sendrequest](https://user-images.githubusercontent.com/73482106/97188311-fcc8fc80-1768-11eb-985b-e868b8d56f72.png)

**Comment by NewKidOnTheBlock** [comment]
Looks like a duplicate of #99

**Comment by DavidXanatos** [comment]
yes looks like, I'll close that one than

**Comment by kiloparseck** [comment]
No, no it's not a duplicate of #99 . In my case nothing crashes, it's just the Internet connection is somehow blocked by SBIE.

**Comment by MrX-at-GitHub** [comment]
I replicate OP attempt and can confirm same issue here on my Windows 8.1 x64
